README.rst

```diff
@@ -190,6 +190,10 @@ Choose from the list of available rules:
    | Ensure there is no code on the same line as the PHP open tag and it is
    | followed by a blank line.
 
+* **blank_line_before_control_statement**
+   | An empty line feed should precede a control statement.
+   | *Rule is: configurable.*
```

Comment:
please remove deprecated fixer from `@Symfony` ruleset and include this one instead